src/Prompt.js

```diff
@@ -60,7 +35,7 @@ class Prompt extends React.Component {
   }
 
   render() {
-    const {smileyDescription, chosenSmiley, value} = this.state
+    const {smileyDescription, chosenSmiley} = this.state
     const smileyScale = [`Terrible`, `Bad`, `Okay`, `Good`, `Great`]
     const smileyId = [`disappointed`, `slightly_frowning_face`, `neutral_face`, `grin`, `satisfied`]
 
```

Comment:
It's probably better to wrap the popup elements in a `div`. Returning an array requires you to add a `key` attribute to each element.